app/yarn.lock

```diff
@@ -7017,7 +7017,7 @@ file-loader@3.0.1, file-loader@^3.0.1:
     loader-utils "^1.0.2"
     schema-utils "^1.0.0"
 
-"file-saver@github:eligrey/FileSaver.js#1.3.8":
+file-saver@eligrey/FileSaver.js#1.3.8:
```

Comment:
Fine for this PR, but can we set a consistent yarn version so that we don't have weird lock-file changes like this?

Example from when we fixed this on the CIVIC repo:
- [issue](https://github.com/hackoregon/civic/issues/728)
- [pr](https://github.com/hackoregon/civic/pull/1071)